### ISSUE TITLE
Is not installing with current flutter

### ISSUE THREAD

**Issue by cvgaviao** [ilikerobots/storyboard#2]
I tried to install it and:
```
The current Dart SDK version is 2.1.2-dev.0.0.flutter-0a7dcf17eb.



Because every version of storyboard depends on recase ^1.0.0+1 which requires SDK version >=1.20.0 <2.0.0, storyboard is forbidden.

So, because weather depends on storyboard ^0.0.3, version solving failed.
pub get failed (1)
exit code 1
```

**Comment by maks** [comment]
PR #3 fixes this, see my comment in the PR on how you can use the fix directly from the PRs git repo for now until its merged and a new version is published to pub.

**Comment by ilikerobots** [comment]
PR #3  is merged and published to 0.0.4.  Thanks @cvgaviao  and @maks.
